addons/containerd/template/testgrid/k8s-ctrd.yaml

```diff
@@ -150,6 +72,10 @@
     minio_object_store_info
     validate_testfile rwtest testfile.txt
     validate_read_write_object_store postupgrade upgradefile.txt
+  unsupportedOSIDs:
+  - rhel-9 # docker is not supported on rhel 9 variants
+  - centos-9 # docker is not supported on rhel 9 variants
+  - ol-9 # docker is not supported on rhel 9 variants
```

Comment:
This seems like an incorrect set of OSIDs
I would expect something like `ol-91` and `rocky-91` to represent oracle linux 9.1 and rocky 9.1

As far as I know we can't test rhel, and centos doesn't exist in versioned 9.x releases

Reply:
updated